src/createGetDataElementValue.js

```diff
@@ -45,7 +41,7 @@ module.exports = function (
     var dataDef = getDataElementDefinition(name);
 
     if (!dataDef) {
-      return undefinedVarsReturnEmpty ? '' : null;
+      return undefinedVarsReturnEmpty ? '' : undefined;
```

Comment:
@dompuiu Double check my thinking here. `getDataElementValue` appears to only be called from `getVar`, but _only when_ `getDataElementDefinition = true`. Therefore, `line 47` really isn't evaluated in normal Turbine flow and only serves as a sanity check for future work.

We're hoping that it means that customers wouldn't be affected by this line change on `line 48`, but we feel `undefined` is a more natural return value when there is no dataElementDefinition but you call this function to retrieve its value.

Reply:
when you say line 47 and 48 are you referring to the old code or the new code? Since the deletion of the isDataElementValuePresent method, it seems the lines have shift.

Reply:
getDataElementValue seems to also be injected into the extension modules: https://developer.adobelaunch.com/extensions/reference/turbine-free-variable/#turbinegetdataelementvaluename-string--

Reply:
Yes, sorry for the confusion. I was talking about line `47` in the old code. 

Great job calling out that we supply `getDataElementValue` to the turbine variable. I think we're going to continue moving forward on switching to `undefined` and will document it as a part of the release notes.